packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -217,6 +232,13 @@ export class TokenRatesController extends BaseController<
       this.update({ contractExchangeRates: {} });
       this.configure({ chainId });
     });
+
+    onPreferencesStateChange(({ useCurrencyRateCheck }) => {
```

Comment:
Instead of adding a new method here, can we just call `this.TokenRatesController.configure` from within the metamask-controller, passing it the appropriate value if the preferences controller state changes?